TheForceEngine/TheForceEngine.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=The Force Engine
+Comment=Reimplementation of the Jedi Engine for Star Wars: Dark Forces and Outlaws
+Comment[de]=Reimplementation der Jedi Engine benutzt von Star Wars: Dark Forces und Outlaws
+Exec=tfelnx
+Type=Application
+Categories=Game;Shooter
+Path=/usr/local/bin
```

Comment:
Again, this will break when another install prefix is specified.  Use cmakes install prefix and let this part be generated by cmake itself.